app/src/main/java/com/example/internetshop/presentation/viewModel/AuthenticationViewModel.kt

```diff
@@ -27,8 +28,11 @@ class AuthenticationViewModel @Inject constructor(
 
     fun getToken() {
         if (username.get().isNullOrEmpty() || password.get().isNullOrEmpty()) {
-            AuthenticationEvent.ToastAuthenticationEvent("Fill in all the fields!")
+            navEventLiveData.value =
+                AuthenticationEvent.ToastAuthenticationEvent("Fill in all the fields!")
```

Comment:
Please change `AuthenticationEvent.ToastAuthenticationEvent` param from string to int and pass string resource id as parameter instead of hardcoded string. For example `AuthenticationEvent.ToastAuthenticationEvent(R.string.error_fill_all_fields)`
You should always use string resources instead of hardcode. This helps you to add localization in your app in future.

Reply:
fixed